tests/components/recorder/test_init.py

```diff
@@ -16,6 +16,7 @@ class TestRecorder(unittest.TestCase):
 
     def setUp(self):  # pylint: disable=invalid-name
         """Setup things to be run when tests are started."""
+        recorder._INSTANCE = None
```

Comment:
Yes, something else must have loaded recorder and not shutdown correctly?

Without this tox fails because of the new check I introduced in the recorder setup to be only a single instance.

Except if setUp happens for each test, but according to me it should be once per test class.